src/snovault/elasticsearch/indexer.py

```diff
@@ -135,13 +135,11 @@ def get_related_uuids(request, es, updated, renamed):
                     {
                         'terms': {
                             'embedded_uuids': updated,
-                            '_cache': False,
```

Comment:
Is there another option here? We probably don't want to worry about cache invalidation when we're finding related uuids for indexing.

Reply:
They have deprecated `_cache` field since 2.0 and have been raising error only after 5.x. It would be hard to cause cache invalidation on-demand during indexing and observe the effect. [More here](https://www.elastic.co/blog/better-query-execution-coming-elasticsearch-2-0)

Reply:
Cool, so this shouldn't change behavior.